composer.json

```diff
@@ -46,6 +46,7 @@
     "laravel/helpers": "^1.4",
     "laravel/passport": "^10.1",
     "laravel/slack-notification-channel": "^2.3",
+    "laravel/socialite": "^5.5",
```

Comment:
Socialite seems like a big add to the project. Given so much work around LDAP and similar integration it seems odd that this would be a feature.